src/Components/modules/Navbar.module.css

```diff
@@ -71,3 +71,17 @@
   letter-spacing: 1.9px;
   text-decoration: none;
 }
+
```

Comment:
- [ ] The same applies in this file too, `.navItem1 and .navItem2` since they share the same styles except for *opacity* and *text-decoration*. you may try to void repetition 🤣✨✨